### DISCUSSION TITLE
Release cycle and end of life for MinIO

### DISCUSSION THREAD

**Discussion by KriAga** [minio/minio#20309] [Q&A] [answered]
Hey Everyone, can someone please help me find the release cycle and end-of-life for particular versions of minIO? 
I cannot find it in any docs, any reference link would be helpful.

**Comment by klauspost** [comment, marked as answer]
We only provide community support of the latest AGPL version. Reach out on `sales@min.io` for commercial options.

**Reply by adriens** [reply]
Hi guys, could we get a dedicated markdown at the root of the repo with this policy ?
